### PR TITLE
[WIP] Fix disable/enable serato metadata checkbox

### PR DESCRIPTION
**Pull request by tech-bash**
Fix: #11226

### CODE REVIEW

**Comment by JoergAtGithub** [comment]
Your PR is based on the Main branch, but this is a patch for the 2.3 branch. Therefore you see all this differences. 
To fix this, you need to rebase your PR: https://github.com/mixxxdj/mixxx/wiki/Using-Git#targeting-another-base-branch

File: src/preferences/dialog/dlgpreflibrary.cpp [hunk 1]
```diff
@@ -417,7 +417,7 @@ void DlgPrefLibrary::slotSeratoMetadataExportClicked(bool checked) {
                             "recommended. Are you sure you want to enable this "
                             "option?"),
                     QMessageBox::Yes | QMessageBox::No) == QMessageBox::No) {
-            checkBox_SeratoMetadataExport->setChecked(false);
+            checkBox_SeratoMetadataExport->setChecked(true);
```
Comment (ronso0) [inline src/preferences/dialog/dlgpreflibrary.cpp:420]: You changed the default state and inverted the state if the user explicitely selected that option.
This is not WIP, this doesn't make any sense at all.

The title of #11226 is "Write Serato Metadata checkbox is ignored when writing track metadata is disabled"
= the state of the Serato checkbox is ignored if the checkbox above ("Write track metadata...") is unchecked
Fix: the GUI should reflect the Serato state is irrelevant
= disable (grey out) the Serato checkbox in that case

**Review by ronso0 — CHANGES_REQUESTED** [review]

File: src/preferences/dialog/dlgpreflibrary.cpp (hunk 1 above)
Comment (tech-bash) [reply src/preferences/dialog/dlgpreflibrary.cpp:420]: Ahhh, got it I was thinking the other way around.

Comment (tech-bash) [reply src/preferences/dialog/dlgpreflibrary.cpp:420]: I should close this PR and open a new one instead.

**Pull request closed without merging**